hack/update-staging-godeps-dockerized.sh

```diff
@@ -51,6 +51,7 @@ while getopts ":df" opt; do
 done
 
 kube::util::ensure_dockerized
+KUBE_GOPATH="/godeps"
```

Comment:
Can we instead support an optional parameter in `kube::golang::setup_env`? I don't like global variable magic. A parameter is explicit.